ansible_collections/arista/avd/roles/eos_designs/templates/designs/l3ls-evpn/overlay/ebgp/router-bgp.j2

```diff
@@ -23,6 +23,20 @@ router_bgp:
         activate: false
 
   address_family_evpn:
+{%     if switch.vtep_ip is arista.avd.defined and ( evpn_hostflap_detection.window is arista.avd.defined or evpn_hostflap_detection.threshold is arista.avd.defined or evpn_hostflap_detection.enabled is arista.avd.defined(false)) %}
+    evpn_hostflap_detection:
+{%         if evpn_hostflap_detection.window is arista.avd.defined %}
+      window: {{ evpn_hostflap_detection.window }}
+{%         endif %}
+{%         if evpn_hostflap_detection.threshold is arista.avd.defined %}
+      threshold: {{ evpn_hostflap_detection.threshold }}
+{%         endif %}
+{%         if evpn_hostflap_detection.enabled is arista.avd.defined(false) %}
+      enabled: 'false'
```

Comment:
Remove quotes. Maybe even refactor so line 34-38 could be replaced with `enabled: {{ evpn_hostflap_detection.enabled | arista.avd.default(true) }}